fission-core/library/Fission/App/Types.hs

```diff
@@ -0,0 +1,64 @@
+module Fission.App.Types
+  ( module Fission.App.Name
+  , App(..)
+  ) where
+
+import           Data.Swagger     hiding (URL, url)
+
+import           Fission.Prelude
+
+import           Fission.App.Name
+import           Fission.URL
+
+
+data App = App
+  { urls       :: [URL]
+  , insertedAt :: UTCTime
+  , modifiedAt :: UTCTime
+  }
+  -- What about deriving Json of normal data?
+  -- deriving (Eq, Generic, Show, ToJSON)
+  deriving (Eq, Show)
+
+
+
+instance FromJSON App where
+  parseJSON = withObject "App" \obj -> do
+    urls       <- obj .: "urls"
+    insertedAt <- obj .: "insertedAt"
+    modifiedAt <- obj .: "modifiedAt"
+
+    return App {..}
+
+instance ToJSON App where
+  toJSON App {..} = object
+    [ "urls"       .= urls
+    , "insertedAt" .= insertedAt
+    , "modifiedAt" .= modifiedAt
+    ]
+
+instance ToSchema App where
+  declareNamedSchema _ = do
+    urls'       <- declareSchemaRef (Proxy :: Proxy [URL])
```

Comment:
Do type applications work for HKT's? like `Proxy @[URL]`?

Reply:
Yup! We had a bunch of `Proxy :: Proxy a`s kicking around initially, and switched to TYpeApplications a while ago, Guess we missed a few! Good catch. So yeah, this can be rewritten `Proxy @[URL]` 👍